audit_log/README.md

```diff
@@ -2,47 +2,66 @@
 
 ## Settings
 
-* `AUDIT_LOG_ORIGIN`, string set to the audit logs origin field
```

Comment:
I wonder if we actually need this README in the future. Maybe just a few paragraphs in the main readme could be enough.

Reply:
it's a good ponder. I think we don't. I'll remove and add some kind of mention in the main